src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -29,14 +29,20 @@ public AddCommand parse(String args) throws ParseException {
         ArgumentMultimap argMultimap =
                 ArgumentTokenizer.tokenize(args, PREFIX_NAME, PREFIX_ID, PREFIX_COUNT, PREFIX_TAG);
 
-        if (!arePrefixesPresent(argMultimap, PREFIX_NAME, PREFIX_ID, PREFIX_COUNT)
+        if ((!arePrefixesPresent(argMultimap, PREFIX_NAME) && !arePrefixesPresent(argMultimap, PREFIX_ID))
                 || !argMultimap.getPreamble().isEmpty()) {
             throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, AddCommand.MESSAGE_USAGE));
         }
 
-        Name name = ParserUtil.parseName(argMultimap.getValue(PREFIX_NAME).get());
-        String id = ParserUtil.parseId(argMultimap.getValue(PREFIX_ID).get());
-        Integer count = ParserUtil.parseCount(argMultimap.getValue(PREFIX_COUNT).get());
+        Name name = arePrefixesPresent(argMultimap, PREFIX_NAME)
+                ? ParserUtil.parseName(argMultimap.getValue(PREFIX_NAME).get())
+                : new Name("dummy name");
+        String id = arePrefixesPresent(argMultimap, PREFIX_ID)
+                ? ParserUtil.parseId(argMultimap.getValue(PREFIX_ID).get())
+                : "999999";
```

Comment:
Can switch to java library `UUID.randomUUID().toString()` for dummy ids.

Reply:
Yup same here as above